README.md

```diff
@@ -466,6 +466,25 @@ export const POST = createRoute(zValidator('form', schema), async (c) => {
 })
 ```
 
+Alternatively, you can use a `_middleware.(ts|tsx)` file in a directory to have that middleware applied to the current route, as well as all child routes. Middleware is ran in the order that it is listed within the array.
+
+An equivilant example to the previous Hono-style middleware configuration is as follows:
+
+```tsx
+// /app/_middleware.tsx
```

Comment:
I know it's a small detail, but wouldn't `/app/_middleware.ts` be better here?

Reply:
fixed!